packages/manager/src/features/Kubernetes/CreateCluster/CreateCluster.tsx

```diff
@@ -244,16 +244,19 @@ export const CreateCluster = () => {
             </StyledDocsLinkContainer>
           </StyledRegionSelectStack>
           <Divider sx={{ marginTop: 4 }} />
-          <Select
-            onChange={(selected: Item<string>) => {
+          <Autocomplete
+            isOptionEqualToValue={(option, value) =>
+              option.value === value.value
+            }
+            onChange={(_, selected: Item<string>) => {
```

Comment:
We don't need an explicit type here. The type of `selected` will be inferred based on the given `options` and `value`

```suggestion
            onChange={(_, selected) => {
```